redi-connect-front/src/components/organisms/RediHeroProgram.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import classnames from 'classnames'
+import { Section, Container, Element, Heading, Content, Columns } from 'react-bulma-components'
+import { useTranslation } from 'react-i18next'
+import SVGImage from '../atoms/SVGImage'
+
+import './RediHeroProgram.scss'
+
+interface Props {
+  type: 'mentor' | 'mentee'
+}
+
+const programIcons = [
+  {
+    name: 'career'
+  },
+  {
+    name: 'clipboard'
+  },
+  {
+    name: 'search'
+  },
+  {
+    name: 'calendar'
+  },
+  {
+    name: 'certificate'
+  }
+]
+
+const RediHeroProgram = ({ type }: Props) => {
+  const { t } = useTranslation()
+  const typeSpecificClass = (className: string) => (classnames(`redi-hero__${className}`, {
```

Comment:
Ok, this you can just replace with one class at the root of the component. You will be then making the component much more easier to read. Since at the end you will be just changing the color which can be managed trough the main class.